src/Microsoft.AspNetCore.Server.Kestrel.Core/RequestBodyTimeout.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core
+{
+    public class RequestBodyTimeout
```

Comment:
Needs xml docs. The API names alone don't provide enough explanation about what these mean and how to set them.

Also, we should provide some validation of inputs, such as MaxTime must be > MinTime, MinimumRate must be >= 0, etc. Consider making the properties readonly and adding a constructor that does these validations.